Simplified/Book/UI/NYPLBookButtonsView.m

```diff
@@ -303,9 +303,21 @@ - (void)updateButtons
       [self.book.defaultAcquisition.availability
        matchUnavailable:nil
        limited:^(NYPLOPDSAcquisitionAvailabilityLimited *const _Nonnull limited) {
+        
+        NSDate *untilDate = self.book.defaultAcquisition.availability.until;
+        NSDate *sinceDate = self.book.defaultAcquisition.availability.since;
+        
+        BOOL hasIncorrectDates = FALSE;
```

Comment:
minor: in objc the convention is to use `NO` instead of FALSE

Reply:
Fixed.